lib/livebook/file_system/s3.ex

```diff
@@ -134,14 +132,18 @@ defmodule Livebook.FileSystem.S3 do
 
   defp put_id(changeset) do
     if bucket_url = get_field(changeset, :bucket_url) do
-      hash = :crypto.hash(:sha256, bucket_url)
-      encrypted_hash = Base.url_encode64(hash, padding: false)
-
-      put_change(changeset, :id, "s3-#{encrypted_hash}")
+      put_change(changeset, :id, id(bucket_url))
```

Comment:
We probably should add the prefix from the corresponding hub at this point?

Reply:
I would say to add a virtual field `:hub_id`, so it would make it easy to add the prefix. WDYT?

Reply:
Sounds good :)